python/setup.py

```diff
@@ -274,7 +273,13 @@ def get_packages(self):
             "fastapi",
             "watchfiles",
         ],
-        "tune": ["pandas", "tensorboardX>=1.9", "requests", pyarrow_dep, "fsspec"],
+        "tune": [
+            "pandas",
+            "tensorboardX>=1.9",
+            "requests",
+            "pyarrow >= 6.0.1",
```

Comment:
bump it uniformly?

Reply:
@matthewdeng and reason we shouldn't bump the minimum PyArrow version for Tune? Kept it as-is to avoid breaking anything

Reply:
I checked Tune code and it seems to only depend on PA `fs` package

Reply:
Let's bump it up